### ISSUE TITLE
ClientHelper field is never used in the clients

### ISSUE THREAD

**Issue by jskeet** [GoogleCloudPlatform/google-cloud-dotnet#419]
We currently don't use the `_clientHelper` field in the `FooClientImpl` that's generated.
I'm about to need it for long-running operations, but that would be better handled by a partial method, e.g.

``` csharp
partial void Initialize(FooSettings settings, ClientHelper clientHelper);
```

That would be called at the _end_ of the constructor (so that the GrpcClient is already set, and doesn't need to be passed in). Most partial classes would never use that.

**Comment by jskeet** [comment]
Chris, now that you're tackling LRO, you've probably got a better idea on this - do we need it? We know we need to expose a more low-level API for getting at settings etc, but presumably this field could go for the moment?

**Comment by jskeet** [comment]
I'm pretty convinced we don't need this now... thoughts on removal?

**Comment by chrisdunelm** [comment]
Yes, I agree this can go. I'll prepare a PR to change the codegen.

**Comment by chrisdunelm** [comment]
Fixed via [toolkit#1017](https://github.com/googleapis/toolkit/pull/1017).
